packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx

```diff
@@ -42,7 +43,7 @@ export class TooltipHost extends BaseComponent<ITooltipHostProps, ITooltipHostSt
   public render() {
     const { calloutProps, content, children, directionalHint, delay } = this.props;
     const { isTooltipVisible } = this.state;
-
+    const tooltipId = getId('tooltip');
```

Comment:
the user might have provided an `id` in props.. we should use that.. so
`const { ......, id } = this.props;`
`const tooltipId = id || getId('tooltip';`